kolibri/plugins/management/assets/src/actions.js

```diff
@@ -24,23 +24,31 @@ const samePageCheckGenerator = require('kolibri.coreVue.vuex.actions').samePageC
  * The methods below help map data from
  * the API to state in the Vuex store
  */
-
-function _userState(data) {
-  // assume just one role for now
-  let kind = UserKinds.LEARNER;
-  if (data.roles.length && data.roles[0].kind === 'admin') {
-    kind = UserKinds.ADMIN;
+function _topUserKind(rolesObjectArray) {
```

Comment:
Broke it out into a separate helper function. Thinking it might be best to make it a local function the same way we did in `user-page/index` since it's not used anywhere else. Let me know what you think.